gateway/modules/crud/sql/read.go

```diff
@@ -248,11 +248,15 @@ func (s *SQL) readexec(ctx context.Context, sqlString string, args []interface{}
 						funcValue, ok := funcMap[functionName]
 						if !ok {
 							// set new function
-							funcMap[functionName] = map[string]interface{}{columnName: value}
+							if columnName == "" {
```

Comment:
Write a comment
// NOTE: This case occurs for count function with no column name (using * operator instead)